modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -198,35 +198,24 @@ const Options = ({
 	});
 
 	useEffect(() => {
-		const localizedOptions = value[editingLanguageId];
+		const availableLanguageIds = Object.getOwnPropertyNames(value);
 
-		if (localizedOptions && localizedOptions.length > 0) {
-			const firstOption = localizedOptions[0];
-
-			if (firstOption.value) {
-				const availableLanguageIds = Object.getOwnPropertyNames(value);
-
-				availableLanguageIds.forEach((languageId) => {
-					normalizedValue[languageId] = value[languageId].map(
-						(option) => {
-							if (option.edited) {
-								return option;
-							}
+		availableLanguageIds.forEach((languageId) => {
+			normalizedValue[languageId] = value[languageId].map((option) => {
+				if (option.edited) {
```

Comment:
Hey @sandrodw3, we have a fake option that is show when we don't have any options created yet. In this case, the option value is empty. Did you consider that when you thought on this solution?

Reply:
Hi @marcosapmf 🙂 thanks for the feedback. I actually considered that. I saw you have always a default option when there is no other options. It's still there. The only difference with my solution is that you will be able to delete it, just as with the other options, but you will always have at least one option. I think it's consistent.

Anyway, I would be grateful if some of you guays can give it a try and tell me if that behavior is ok for you, or you need any change.

Thanks again! 🙂

Reply:
Hey @sandrodw3, thanks for the clarification. 

We'll do that!

Thanks to you